src/common/logger.ts

```diff
@@ -63,6 +63,24 @@ export const getLogger = (name: string) => {
   return newLogger
 }
 
+const auditLogger = getLogger('audit-log')
```

Comment:
So audit logs will be mix with other types of logs? What can we do for the performance of lambda-handler?

Reply:
As audit log pushed to RDS is for analytic usage,      . the concern on lambda is not the speed but  whether it can finish processing 1 hour rotation log file in 15 mins.

Reply:
RDS -> bigquery

Reply:
Shall we do a simple stress test?